model/src/w3adatmd.F90

```diff
@@ -2180,15 +2161,17 @@ SUBROUTINE W3XDMA  ( IMOD, NDSE, NDST, OUTFLAGS )
       ALLOCATE ( WADATS(IMOD)%XTAUOCY(1), STAT=ISTAT )
       CHECK_ALLOC_STATUS ( ISTAT )
     END IF
-#ifdef W3_CESMCOUPLED
     IF ( OUTFLAGS( 6, 14) ) THEN
```

Comment:
This is adding a new output variable, but I see no mention of it in the description of the PR or in the model/inp or nml variables.